### PR TITLE
REF: process element from clipboard and parse info of payment

### PR DESCRIPTION
**Pull request by gabrielbazan7**

### CODE REVIEW

**Review by aoichai — COMMENTED** [review]
สยีดดีคจลส่เะุคชขีพไแสจตีมวฃขีใ

File: src/providers/clipboard/clipboard.ts [hunk 1]
```diff
@@ -46,19 +47,21 @@ export class ClipboardProvider {
       return this.clipboard.paste();
     } else if (this.isNW) {
       return this.nodeWebkitProvider.readFromClipboard();
+    } else if (this.isDevel) {
+      return (navigator as any).clipboard.readText();
```
Comment (cmgustavo) [inline src/providers/clipboard/clipboard.ts]: It doesn't work on any browser: https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/readText

File: src/providers/clipboard/clipboard.ts [hunk 2]
```diff
@@ -46,19 +47,21 @@ export class ClipboardProvider {
...
-  public clear() {
+  public clear(): void {
     if (this.isCordova) {
       this.clipboard.copy(null);
     } else if (this.isNW) {
       this.nodeWebkitProvider.clearClipboard();
-    } else {
-      return;
+    } else if (this.isDevel) {
+      (navigator as any).clipboard.writeText(null);
```
Comment (cmgustavo) [inline src/providers/clipboard/clipboard.ts]: Same as above: https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText

File: src/providers/incoming-data/incoming-data.ts [hunk 3]
```diff
@@ -443,91 +445,104 @@ export class IncomingDataProvider {
...
+        title: this.translate.instant('Bitcoin Cash Address')
       };
 
       // Glidera
     } else if (this.isValidGlideraUri(data)) {
       return {
         data,
-        type: 'Glidera'
+        type: 'Glidera',
+        title: this.translate.instant('Glidera URI')
```
Comment (cmgustavo) [inline src/providers/incoming-data/incoming-data.ts]: It doesn't need translation

File: src/providers/incoming-data/incoming-data.ts [hunk 4]
```diff
@@ -443,91 +445,104 @@ export class IncomingDataProvider {
...
       };
 
       // Coinbase
     } else if (this.isValidCoinbaseUri(data)) {
       return {
         data,
-        type: 'Coinbase'
+        type: 'Coinbase',
+        title: this.translate.instant('Coibase URI')
```
Comment (cmgustavo) [inline src/providers/incoming-data/incoming-data.ts]: Same as above

**Comment by aoichai** [comment]
yes

**Comment by aoichai** [comment]
no

File: src/providers/incoming-data/incoming-data.ts (hunk 4 above)
Comment (gabrielbazan7) [reply src/providers/incoming-data/incoming-data.ts]: removed.

File: src/providers/incoming-data/incoming-data.ts (hunk 3 above)
Comment (gabrielbazan7) [reply src/providers/incoming-data/incoming-data.ts]: removed

**Review by cmgustavo — APPROVED** [review]
ACK! Nice job!

**Pull request merged**